scene/resources/style_box_flat.cpp

```diff
@@ -310,8 +310,36 @@ inline void set_corner_scale(const Rect2 &style_rect, const Rect2 &inner_rect, c
 	}
 }
 
+void StyleBoxFlat::set_corner_smoothing_all(real_t smoothing) {
+	for (int i = 0; i < 4; i++) {
+		corner_smoothing[i] = smoothing;
+	}
+
+	emit_changed();
+}
+
+void StyleBoxFlat::set_corner_smoothing_individual(const real_t smoothing_top_left, const real_t smoothing_top_right, const real_t smoothing_bottom_right, const real_t smoothing_bottom_left) {
+	corner_smoothing[0] = smoothing_top_left;
+	corner_smoothing[1] = smoothing_top_right;
+	corner_smoothing[2] = smoothing_bottom_right;
+	corner_smoothing[3] = smoothing_bottom_left;
+
+	emit_changed();
+}
+
+void StyleBoxFlat::set_corner_smoothing(const Corner p_corner, real_t p_corner_smoothing) {
+	corner_smoothing[p_corner] = p_corner_smoothing;
```

Comment:
```suggestion
	ERR_FAIL_INDEX((int)p_corner, 4);
	if (corner_smoothing[p_corner] == p_corner_smoothing) {
		return;
	}
	corner_smoothing[p_corner] = p_corner_smoothing;
```
Or this will crash with incorrect input, also early return to avoid unnecessary emitting of `changed`